server.ts

```diff
@@ -126,6 +126,23 @@ async function prepareDataForDB(data, properties) {
     return out
 }
 
+async function initHelloWorldGit() {
+
+    const connection = await db.connect()
+    const helloWorldGit = await connection.getRepository(Git).findOne('hello_world')
+    if (helloWorldGit == undefined) {
+    const git = {
+    id: 'hello_world',
+    address: `https://github.com/cybergis/cybergis-compute-hello-world.git`,
+    isApproved: true,
+    createdAt: new Date(),
+    updatedAt: new Date()
+    }
+    await connection.getRepository(Git).save(git) 
+    }}
```

Comment:
I don't think this is well formatted. If you are using VS Code, you can just type Shift + Option + F to format the code.

Reference: https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode

Reply:
How does it look now ? @zimo-xiao It reformatted a large part of the codebase.